src/MatrixTypes_Trilinos.f90

```diff
@@ -262,7 +262,7 @@ SUBROUTINE setRow_TrilinosMatrixType(matrix,i,j,setval)
       REAL(SRK),INTENT(IN) :: setval(:)
 
       REQUIRE(matrix%isInit)
-      REQUIRE(((j <= matrix%n) .AND. (i <= matrix%n)) .AND. ((j > 0) .AND. (i > 0)))
+      REQUIRE(((ALL(j <= matrix%n)) .AND. (ALL(i <= matrix%n))) .AND. ((ALL(j > 0)) .AND. (ALL(i > 0))))
```

Comment:
so I think the ALL additions around the i components will be a problem since it's just a scalar value:

```
Error: 'mask' argument of 'all' intrinsic at (1) must be a logical array
```

The ones around the j terms should be fine.  It should roughly match the DBC check in setRow_SparseMatrixType.

Reply:
Yep, i is just a scalar.  I fixed it.